packages/engine/tests/units/globals/immutable_from_behaviors/integration-test.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "steps": 2,
+    "expected-output": {
+      "json-state": {
+        "1": [
```

Comment:
It _shouldn't_ matter, but perhaps we can check the globals the next step too, just to make sure it stayed the same